src/app/routes/auth/containers/Forgot.js

```diff
@@ -1,125 +1,106 @@
-                    <header>
-                      Forgot Password
-                    </header>
 
+    constructor() {
+        super();
+        // Initialize variable for forgot password action method to keep form as a controlled component
+        this.formSubmit = this.forgotPasswordAction.bind(this);
+        this.form = this.passwordRecoveryForm();
+    }
+
+    componentDidMount() {
+        // Initialize variables for access to form fields
+        this.email = ReactDOM.findDOMNode(this.refs.email);
+    }
+
+    forgotPasswordAction(event) {
+        event.preventDefault();
+
+        const formValid = this.email.value.length > 0 && this.email.validity.valid;
+        if (formValid) {
+            // Todo: Send password recovery email here
+            // email value -> this.email.value
+        }
+    }
+
+    passwordRecoveryForm() {
+        return(
+            <UiValidate options={validationOptions}>
+                <form action="#/login" id="login-form" onSubmit={this.formSubmit} className="smart-form client-form">
+                    <header>Forgot Password</header>
                     <fieldset>
-
-                      <section>
-                        <label className="label">Enter your email address</label>
-                        <label className="input"> <i className="icon-append fa fa-envelope"/>
-                          <input type="email" name="email"/>
-                          <b className="tooltip tooltip-top-right"><i className="fa fa-envelope txt-color-teal"/> Please
-                            enter email address for password reset</b></label>
-                      </section>
-                      <section>
-										<span className="timeline-seperator text-center text-primary"> <span className="font-sm">OR</span>
-                            </span></section>
-                      <section>
-                        <label className="label">Your Username</label>
-                        <label className="input"> <i className="icon-append fa fa-user"/>
-                          <input type="text" name="username"/>
-                          <b className="tooltip tooltip-top-right"><i className="fa fa-user txt-color-teal"/> Enter your
-                            username</b> </label>
-                        <div className="note">
-                          <a href="#/login">I remembered my password!</a>
-                        </div>
-                      </section>
-
+                        <section>
+                            <label className="label">Enter your email address</label>
+                            <label className="input"> <i className="icon-append fa fa-envelope"/>
+                            <input type="email" name="email" ref="email"/>
+                            <b className="tooltip tooltip-top-right"><i className="fa fa-envelope txt-color-teal"/>
+                                Please enter email address for password reset</b></label>
```

Comment:
let's try this --> "Need to recovery your account? Please enter your email address."